MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -5786,6 +5786,23 @@ public string GetProgressiveAudioFullCommandLine(EncodingJobInfo state, Encoding
                 }
             }
 
+            // Copy the movflags from GetProgressiveVideoFullCommandLine
+            // See #9248 and the associated PR for why this is needed
+            var mp4ContainerNames = new HashSet<string>
+            {
+                "mp4",
+                "m4a",
+                "m4p",
+                "m4b",
+                "m4r",
+                "m4v",
+            };
```

Comment:
This should be moved to the top and be `static readonly`

You should also initialize the hashset with a comparer instead of calling `.ToLower()`
```
new HashSet<string>(StringComparer.OrdinalIgnoreCase)
```

Reply:
Resolved, that's a pretty nice C# feature :)